tests/modules/cnvkit/main.nf

```diff
@@ -2,18 +2,29 @@
 
 nextflow.enable.dsl = 2
 
-include { CNVKIT } from '../../../modules/cnvkit/main.nf' addParams( options: [ 'args': '--output-reference reference.cnn' ] )
+include { CNVKIT }               from '../../../modules/cnvkit/main.nf' addParams( options: [ 'args': '--output-reference reference.cnn' ] )
+include { CNVKIT as CNVKIT_WGS } from '../../../modules/cnvkit/main.nf' addParams( options: [ 'args': '--output-reference reference.cnn --method wgs' ] )
 
-workflow test_cnvkit {
+workflow test_cnvkit_hybrid {
     tumourbam = file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
     normalbam = file(params.test_data['sarscov2']['illumina']['test_single_end_sorted_bam'], checkIfExists: true)
-    
+
     input = [ [ id:'test' ], // meta map
-              tumourbam, 
-              normalbam 
+              tumourbam,
+              normalbam
             ]
     fasta   = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
     targets = file(params.test_data['sarscov2']['genome']['baits_bed'], checkIfExists: true)
 
     CNVKIT ( input, fasta, targets )
 }
+
+workflow test_cnvkit_wgs {
+    input = [ [ id:'test'], // meta map
+              [ file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam'], checkIfExists: true) ],
```

Comment:
I guess this module will also be part of the sarek copy number calling sub workflow? In that case it needs to also function with cram files. Could you add a test for that?

Reply:
Ok, [done](https://github.com/fbdtemme/modules/blob/07fa77109dba21ed57119677332e2cac418e65f1/tests/modules/cnvkit/main.nf#L33-L41).